### ISSUE TITLE
#1656892060------------- ARIA toggle fields must have an accessible name

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#465]
<strong>Source:</strong>`<div aria-checked="false" aria-label="" class="toggle" role="checkbox" tabindex="0"><svg aria-hidden="true" class="check-svg" viewBox="0 0 16 16"><path d=""></path></svg><slot></slot></div>`
<strong>Element Location: </strong>[".sc-ePDLzJ.iBkgkw:nth-child(1) > td:nth-child(1) > calcite-checkbox[data-test=\"row-selection-checkbox\"]",".toggle[role=\"checkbox\"][aria-label=\"\"]"]
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@7e38e254
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
